ghost/core/core/server/data/schema/default-settings/default-settings.json

```diff
@@ -565,5 +565,15 @@
             },
             "type": "number"
         }
+    },
+    "recommendations": {
+        "recommendations_enabled": {
+            "defaultValue": "true",
```

Comment:
```suggestion
            "defaultValue": "false",
```

Reply:
Updated, following the Slack discussion [here](https://ghost.slack.com/archives/C04TMVA1D7A/p1693310637853019)